VTUL/dlp-access
# Allows site admin to upload podcast episode files and to create new Archive records to store the associated metadata

**Allows site admin to upload podcast episode files and to create new Archive records to store the associated metadata.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2097) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Allows site admin to upload podcast episode files and to create new Archive records to store the associated metadata.

# What's the changes? (:star:)
* Adds form to allow site admin to:
  * Upload podcast episode files (audio, thumbnail image, and captions)
  * Specify metadata for the new podcast
* Creates Archive record based on the metadata and and uploaded files
* Associates the Archive record with the Podcast collection

# How should this be tested?
* Switch amplify environment to `podcastdeposit`
* Start server with REACT_APP_MINT_API_KEY env variable set so that a noid can be minted for the Archive
* Visit the `/podcastDeposit` page
* Login with your dlp-access credentials (must be admin)
* Fill in metadata fields and upload podcast file(s). The audio file is required, others are optional.
* Check that your files were successfully uploaded to s3.
* After you click Submit you'll be redirected to the view page. Check that the displayed metadata is accurate.

# Additional Notes:
* app branch: `podcast_deposit`
* amplify env: `podcastdeposit`

# Interested parties
@yinlinchen 

(:star:) Required fields
